GitUI/Interops/DwmApi/DwmApi.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace GitUI.Interops.DwmApi
+{
+    /// <summary>
+    /// Allow to set dark title bar on Win10. From: https://stackoverflow.com/questions/57124243/winforms-dark-title-bar-on-windows-10/62811758#62811758
+    /// </summary>
+    internal static class DwmApi
+    {
+        [DllImport("dwmapi.dll")]
+        private static extern int DwmSetWindowAttribute(IntPtr hwnd, uint attr, ref int attrValue, int attrSize);
+
+        private const uint DWMWA_USE_IMMERSIVE_DARK_MODE_BEFORE_20H1 = 19;
+        private const uint DWMWA_USE_IMMERSIVE_DARK_MODE = 20;
+
+        internal static bool UseImmersiveDarkMode(IntPtr handle, bool enabled)
+        {
+            if (!IsWindows10OrGreater(17763))
+            {
+                return false;
+            }
+
+            var attribute = IsWindows10OrGreater(18985)
+                ? DWMWA_USE_IMMERSIVE_DARK_MODE
+                : DWMWA_USE_IMMERSIVE_DARK_MODE_BEFORE_20H1;
+
+            int useImmersiveDarkMode = enabled ? 1 : 0;
+            return DwmSetWindowAttribute(handle, attribute, ref useImmersiveDarkMode, sizeof(int)) == 0;
+        }
+
+        private static bool IsWindows10OrGreater(int build = -1) => Environment.OSVersion.Version.Major >= 10 && Environment.OSVersion.Version.Build >= build;
```

Comment:
This default is not used anywhere, so remove it.

```suggestion
        private static bool IsWindows10OrGreater(int build) => Environment.OSVersion.Version.Major >= 10 && Environment.OSVersion.Version.Build >= build;
```